### DISCUSSION TITLE
Sepformer results in WHAM! without any augmentation

### DISCUSSION THREAD

**Discussion by JusperLee** [speechbrain/speechbrain#1711] [Q&A]
I noticed that you only report the results of WHAM! with data augmentation, why not report the results without augmentation?

Do you have unenhanced results, can you publish them?

**Comment by JusperLee** [comment]
@popcornell @mravanelli

**Comment by JusperLee** [comment]
@ycemsubakan

**Comment by ycemsubakan** [comment]
We will update the numbers on the readme file once we update the long paper on arxiv. Thanks for your message @JusperLee .

**Reply by JusperLee** [reply]
Hello, I would like to ask if it can be made public in advance? I will add citations to your paper after it is published.
